source/leuven/genius_2_rocky.rst

```diff
@@ -1,121 +1,267 @@
+   -------------------------------------
+     For detailed information about a specific "CP2K" package (including how
+     to load the modules) use the module's full name.
+     Note that names that have a trailing (E) are extensions provided by other
+     modules. For example:
+        $ module spider CP2K/8.2-intel-2021a
+   -------------------------------------
+
+As suggested as part of the output, you can obtain more information about one
+of the available versions of the CP2K module by executing:
+
+.. code-block:: shell
+
+   $ module spider CP2K/8.2-intel-2021a
+
+   -------------------------------------
+     CP2K: CP2K/8.2-intel-2021a
+   -------------------------------------
+       Description:
+         CP2K is a freely available (GPL) program, ...
+
+
+    You will need to load all module(s) on any one of the lines below before
+    the "CP2K/8.2-intel-2021a" module is available to load
+
+      cluster/genius/amd
+      cluster/genius/amd_long
+      cluster/genius/batch
+      ...
+      cluster/wice/batch
+      ...
+
+This command shows which cluster modules will make the ``CP2K/8.2-intel-2021a``
+module available. As discussed earlier, loading ``cluster/wice/batch`` is one
+example of a cluster module that suffices to make ``CP2K/8.2-intel-2021a``
+available. For more information about ``module spider``, have a look at the
+`Lmod documentation page <https://lmod.readthedocs.io/en/latest/135_module_spider.html>`__
 
+.. note::
 
-.. _impact_on_starting_jobs:
+   In contrast to previous behavior, modules from different toolchain versions
+   will now be available automatically. On Genius, all modules since 2018a
+   will be available, and on wICE, all modules starting from 2021a. For a few
+   legacy modules, installation is impossible on a recent operating system. In
+   such a case, it is recommended to use a replacement module from a newer
+   toolchain version.
 
-Impact on starting jobs on Genius and wICE
-------------------------------------------
-In order to minimize the changes you need to make to your jobscripts, an appropriate module path (``$MODULEPATH``) will be set by default at the start of your job of the the migration of Genius to Rocky 8 OS. This new module path will now contain all toolchain versions, so all the modules, starting from 2018a on Genius and starting from 2021a on wICE. This is an important change! Previously the modulepath was set to a single toolchain, that would not change over time. You might have set a module path in your jobscripts to refer to newer toolchains. Module use/unuse should now only be needed in exceptional cases.
+.. _manually_modifying_modulepath:
 
-.. note::
+Manually modifying the modulepath
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-   If you have set a module path explicitly in your jobscript, you can remove it from your jobscript or change it to the module path for Rocky 8.
+As discussed in the previous section, the recommended approach to set your
```

Comment:
Can you please rephrase this sentence, or even better, split it up into two sentences?

Reply:
As discussed in the previous section, the recommended approach to set your $MODULEPATH environment variable is by using the cluster module. This will make modules from the correct software stack are available,.